Refactor system card content for clarity and updates

Omdhm opened the pull request:
Removed operating context details and supporting documentation. Updated risk and opportunity sections with stakeholder adjustments. Enhanced clarity on mitigated and unmitigated risks.

Omdhm committed:
Refactor system card content for clarity and updates

Removed operating context details and supporting documentation. Updated risk and opportunity sections with stakeholder adjustments. Enhanced clarity on mitigated and unmitigated risks.

aruss175 commented on system_card.md:
```diff
@@ -23,25 +23,12 @@ Use judgment and review debugging figures to assess performance.
-<br>
-
 ## Risk
 In the following tables, the *risk factor category* (*RC*) is taken from [RM100 R-105908](https://aris.equinor.com/#default/item/c.requirement.Production.hholoN3kEeRmCQBQVrsUrw.-1) with the following abbreviations
```
you should clarify that RM100 is an equinor internal governing document and remove the hyperlink.

aruss175 commented on system_card.md:
```diff
@@ -60,31 +47,31 @@ In the following tables, the *risk factor category* (*RC*) is taken from [RM100
 | T  | More accurate, standardized DCA in forecasting for engineers | Improved decision-making, optimized production strategies | Reservoir Engineers, Production Engineers |
 | H  | Increased job satisfaction for engineers due to reduced manual workload | Higher morale, better retention rates | Reservoir Engineers, Production Engineers |
-| T  | Traceability of forecast inputs and logging of well forecast inputs | Enhanced auditability and traceability of forecasts  | DSA, IT Ops, Product Governance |
+| T  | Traceability of forecast inputs and logging of well forecast inputs | Enhanced auditability and traceability of forecasts  |  Product Governance |
```
Product Governance should be "Equinor-internal product governance"

aruss175 commented on system_card.md:
```diff
@@ -60,31 +47,31 @@ In the following tables, the *risk factor category* (*RC*) is taken from [RM100
 | T  | Easier implementation and adoption due to integrating with existing commercial software (e.g., pForecast) | Reduced integration complexity | Reservoir Engineers, Production Engineers |
-| T  | Scalable cloud architecture| Reduces computational bottlenecks | DSA, IT Ops, AI Platform teams |
-| P  | Clear governance and architecture compliance (AC‑1526, TRL4) | Smoother scaling and enterprise adoption | Product governance, Architecture team, DSA |
+| T  | Scalable cloud architecture| Reduces computational bottlenecks | Architecture Team|
```
Equinor Internal Architecture team

aruss175 commented on system_card.md:
```diff
@@ -60,31 +47,31 @@ In the following tables, the *risk factor category* (*RC*) is taken from [RM100
-| T  | Scalable cloud architecture| Reduces computational bottlenecks | DSA, IT Ops, AI Platform teams |
-| P  | Clear governance and architecture compliance (AC‑1526, TRL4) | Smoother scaling and enterprise adoption | Product governance, Architecture team, DSA |
+| T  | Scalable cloud architecture| Reduces computational bottlenecks | Architecture Team|
+| P  | Clear governance and architecture compliance (AC‑1526, TRL4) | Smoother scaling and enterprise adoption | Product governance, Architecture team |
```
(Equinor-Internal Processes: AC-1526, TRL4)

aruss175 commented on system_card.md:
```diff
@@ -60,31 +47,31 @@ In the following tables, the *risk factor category* (*RC*) is taken from [RM100
 | RC | Description                                      | Consequence           | Mitigating Action                                      | Stakeholders |
 |:--:|:-------------------------------------------------|:----------------------|:-------------------------------------------------------|:-------------|
-| T  | Code maintainability       | Delays in bug fixes, difficulty adding features | Modular architecture; CI/CD with GHAS (CodeQL, Dependabot); governance via AC‑1526 | DSA, IT Ops, Product Governance |
+| T  | Code maintainability       | Delays in bug fixes, difficulty adding features | Modular architecture; CI/CD with GHAS (CodeQL, Dependabot); governance via AC‑1526 | IT Team, Product Governance |
```
Equinor Internal IT Team

aruss175 commented on system_card.md:
```diff
@@ -60,31 +47,31 @@ In the following tables, the *risk factor category* (*RC*) is taken from [RM100
-| T  | Inability to rollback to previous model and predictions | Prolonged outages, incorrect forecasts | Version control for models and configurations; automated rollback procedures | DSA, IT Ops |
-| T  | Security: DSA App Registration & Subscription Keys | Unauthorized access, data breaches | Implement strict access controls; regular security audits; use of managed identities | IT Security, DSA |
-| T  | IT Operations Cost OverRun | Budget overruns, resource constraints | Implement cost monitoring and optimization strategies; regular budget reviews | IT Ops, DSA |
+| T  | Data quality and availability issues in Production Data Mart | Inaccurate forecasts, system failures | Data validation pipelines; integration with PDM; fallback to manual workflows | PDM Team |
```
Equinor-Internal PDM Team

aruss175 commented on system_card.md:
```diff
@@ -60,31 +47,31 @@ In the following tables, the *risk factor category* (*RC*) is taken from [RM100
+| T  | Inability to rollback to previous model and predictions | Prolonged outages, incorrect forecasts | Version control for models and configurations; automated rollback procedures | IT team |
+| T  | Security: DSA App Registration & Subscription Keys | Unauthorized access, data breaches | Implement strict access controls; regular security audits; use of managed identities | IT Security |
+| T  | IT Operations Cost OverRun | Budget overruns, resource constraints | Implement cost monitoring and optimization strategies; regular budget reviews | IT team |
 
```
All of the lines above should say "Equinor Internal IT Team"

aruss175 commented on system_card.md:
```diff
@@ -23,25 +23,12 @@ Use judgment and review debugging figures to assess performance.
 
 <br>
 
-## Supporting Documentation
```
i think it is ok to keep the supporting documentation section, but rename it "Equinor-Specific Supporting Documentation" with the only line underneath being the CI number.  "Please use CI number 121455 to find associated documentation"

Omdhm committed:
Revise version details and internal references

Updated version information and clarified references to Equinor's internal processes. Added Equinor-internal specifications for stakeholders and documentation.

aruss175 commented:
Looks good from me Oumou!  Would like @AlirezaNaziri to take a look before approval.

AlirezaNaziri commented:
@aruss175 

Opportunity (Upside Risk):
RC: T
Generalization method could potentially decrease risk of highly sensitive of Decline Curve to outliers and noise in production data. 
Consequence: improve decline curve fitting
stakeholders: Reservoir Engineers, Production Engineers

Unmitigated Risk:
RC: T
Curve fitting may yield non-physical sense
Consequence: Decreasing forecast accuracy
Suggested Mitigation: Validate model outputs against known reservoir behavior or reservoir simulators (Eclipse) results.
Stakeholders: Reservoir Engineers, Production Engineers
